packages/create-cloudflare/templates/hello-world/ts/src/index.ts

```diff
@@ -23,6 +23,9 @@ export interface Env {
 	//
 	// Example binding to a Queue. Learn more at https://developers.cloudflare.com/queues/javascript-apis/
 	// MY_QUEUE: Queue;
+	//
```

Comment:
This addition seems fine since there's other examples of it here, but now that I'm looking at it I think it would be better if we convince users to run `wrangler types` to generate this instead of updating the type manually.